src/types.d.ts

```diff
@@ -1,56 +1,75 @@
 
-declare module '@google-cloud/common' {
-  import * as request from 'request';
-
-  type LogFunction = (message: any, ...args: any[]) => void;
-
-  export interface Logger {
-    error: LogFunction;
-    warn: LogFunction;
-    info: LogFunction;
-    debug: LogFunction;
-    silly: LogFunction;
-  }
-
-  export interface LoggerOptions {
-    level?: string;
-    levels?: string[];
-    tag?: string;
-  }
-
-  export const logger: {
-    (options?: LoggerOptions | string): Logger;
-    LEVELS: string[];
-  };
+import * as request from 'request';
+
+export interface Common {
+  Service: Service;
+  logger: Logger;
+}
+/**
```

Comment:
I think the copyright header must be at the top.

Reply:
Good catch.

Reply:
Done